client/src/components/Ratings/index.jsx

```diff
@@ -0,0 +1,228 @@
+/**
```

Comment:
lost semantic value when renaming this file
instead have default export in index.js that exports Ratings.jsx